frontend/src/store/index.js

```diff
@@ -39,6 +40,10 @@ const store = new Vuex.Store({
   getters: {
     [IS_ONLINE]: state => state.isOnline,
     [UNSYNCRONISED_DRIVES]: state => state.unsyncedDrives,
+    [UNSYNCRONISED_DRIVES_TOTAL_MILEAGE]: state => state.unsyncedDrives.reduce(
```

Comment:
Why is the reduce needed twice? It is the same function so should be extracted.
Here getter is be polluting the global space- can't it be more local to the component?

Reply:
IMHO the purpose of the redux selectors is to derive the piece of state to the component and if some additional calculations are necessary they should be done by selector than in component.